libkbfs/backpressure_disk_limiter.go

```diff
@@ -1053,10 +1053,9 @@ func (bdl *backpressureDiskLimiter) reserveBytes(
 	if count < 0 {
 		return count, nil
 	}
-	// We calculate the total free bytes by adding the reported free bytes and
-	// the non-`tracker` used bytes.
-	tracker.updateFree(freeBytes + bdl.overallByteTracker.used -
-		tracker.usedResources())
+	// TODO: verify there's not any other kind of bytes we care about
+	// subtracting too.
+	tracker.updateFree(freeBytes)
```

Comment:
Are you sure you want to do this? This will affect the working set block cache too, such that it'll now be tracking global free space. It throws off the fraction calculation.

Reply:
I think so?

My intent is to let the sync block cache take over the entire disk if it wants to and starve the working set cache, as discussed on https://keybase.atlassian.net/browse/KBFS-3517?focusedCommentId=59212&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-59212

But I am not confident that I fully understand all of the block cache machineries.

Reply:
maybe it would be better to do it the old way if the disk percentages add up to less than 100%, and this way if they add up to more.

Reply:
I would have expected to have seen something that finds the places that updates the `overallByteTracker`, and NOT update it when the type of cached bytes is for the sync cache.

Reply:
Actually, I guess you could get rid of `overallByteTracker` completely, and just rely on the working set cache in those places that use it?

Reply:
Hmm. I was hoping to keep the `overallByteTracker` code in place, in case we ever get other disk caches that need to compete for space.

Reply:
I'll think about it and write back here.

Reply:
Fine with me.  Then the first thing I said.  The sync cache shouldn't update the overall tracker if it shouldn't count towards the total bytes used by KBFS.

Reply:
@jzila are you still thinking about it or should I go with strib's most recent suggestion?

Reply:
Go ahead with strib's most recent suggestion.